tasks.py

```diff
@@ -538,17 +539,13 @@ def type_schema(
                 schema_path = schema_dir.joinpath(f"{model.__name__}.json")
                 json_str: str = model.schema_json(indent=indent) + "\n"
 
-                if issubclass(model, Datasource):
-                    print(f"🙈  {name} - is a Datasource; skipping")
-                    continue
-
                 if schema_path.exists():
                     if json_str == schema_path.read_text():
                         print(f"✅  {name} - {schema_path.name} unchanged")
                         continue
 
                 schema_path.write_text(json_str)
-                print(f"✅  {name} - {schema_path.name} schema updated")
+                print(f"🔃  {name} - {schema_path.name} schema updated")
```

Comment:
Changed the symbol here so it's easier to tell at-a-glance if schemas have changed